provisioning/plugins/corim-psa-decoder/ccaplatformconfigid.go

```diff
@@ -33,19 +33,23 @@ func (o *CCAPlatformConfigID) FromMeasurement(m comid.Measurement) error {
 	return nil
 }
 
-func (o CCAPlatformConfigID) MakeRefAttrs(c PSAClassAttributes) (*structpb.Struct, error) {
+func (o CCAPlatformConfigID) GetRefValType() string {
+	return "platform-config"
+}
+
+func (o CCAPlatformConfigID) MakeRefAttrs(c ClassAttributes, scheme string) (*structpb.Struct, error) {
```

Comment:
We do not scheme sctring here so just hard code to cca and use it to avoid confusion

Reply:
Agreed to add a comment and clarify why this is needed!

Reply:
Clarifying comment added!